packages/base64/test/bench-main.js

```diff
@@ -0,0 +1,46 @@
+import { encodeBase64, decodeBase64 } from '../src/main';
```

Comment:
For files in test* that are not run by `yarn test` I always include a comment at the beginning explaining their purpose -- that they are simply meant to be run manually in order for a human to look at the output. Or does this one get run automatically somehow? What does CI do with it?

Reply:
Is this use case "monkey testing"? Nice to have a term for it.

Reply:
Adding a description. I’m electing not to add an abstraction for the boilerplate. Even with the inner loop in-tact, the overhead of a benchmark abstraction adds 10% error.

Reply:
ok. Resolving.

Reply:
Doesn’t seem to match https://en.wikipedia.org/wiki/Monkey_testing